src/main/resources/hudson/plugins/active_directory/ActiveDirectorySecurityRealm/help-removeIrrelevantGroups.html

```diff
@@ -3,4 +3,5 @@
     This can significantly improve performance in environments with a large number of groups
     but a small number of corresponding rules defined by the Authorization Strategy.
     The user is required to re-login in order for the changes to take effect.
+    <br><strong>Note:</strong> this setting may be incompatable with some Authorzation Strategies.
```

Comment:
@recena, you've got a couple typos in your recommended version, corrected below:

    <p>
      <strong>Note:</strong> this setting may be incompatible with some Authorization Strategies.
    </p>